examples/user_texture/main.rs

```diff
@@ -1841,24 +1841,31 @@ impl App {
             self.device.end_command_buffer(command_buffer)?;
 
             // Submit command
-            self.device.queue_submit(
-                self.graphics_queue,
-                &[vk::SubmitInfo::builder()
-                    .wait_dst_stage_mask(&[vk::PipelineStageFlags::COLOR_ATTACHMENT_OUTPUT])
-                    .command_buffers(&[command_buffer])
-                    .wait_semaphores(&[self.image_available_semaphores[self.current_frame]])
-                    .signal_semaphores(&[self.render_finished_semaphores[self.current_frame]])
-                    .build()],
-                fence,
-            )?;
+            {
+                let wait_dst_stage_mask = [vk::PipelineStageFlags::COLOR_ATTACHMENT_OUTPUT];
+                self.device.queue_submit(
+                    self.graphics_queue,
+                    &[vk::SubmitInfo::builder()
```

Comment:
Again from_ref might help to remove .build(). Constructing slices inside the same expression is fine.